test/unit/api.spec.js

```diff
@@ -0,0 +1,20 @@
+import postscribe from '../../dist/postscribe';
+
+describe('api', function() {
+  const itCalls = name => {
+    it(`calls ${name}`, function(done) {
+      this.timeout(1000);
+      $(document).ready(() => {
+        const options = {done};
+        options[name] = () => {
+          expect(1).to.be.ok();
```

Comment:
But you're never doing a negative check... This is actually a perfect use case for sinon mocks.